dotenv-example

```diff
@@ -39,6 +39,12 @@ DISCORD_NOTIFY_GROUP_RYZEN5600=
 DISCORD_NOTIFY_GROUP_RYZEN5800=
 DISCORD_NOTIFY_GROUP_RYZEN5900=
 DISCORD_NOTIFY_GROUP_RYZEN5950=
+DISCORD_NOTIFY_GROUP_RYZEN7800X3D=
+DISCORD_NOTIFY_GROUP_RYZEN9800X3D=
+DISCORD_NOTIFY_GROUP_RYZEN9600=
```

Comment:
Also can I make a nit about these? Are the models 9600 or 9600X? I think the same applies for the other models.

Reply:
Ah good catch they are x variants, not non x. I'll fix that when I'm at my desk next